### ISSUE TITLE
adapt-contrib-languagePicker: Missing aria-label in navigation button

### ISSUE THREAD

**Issue by brian-learningpool** [adaptlearning/adapt_framework#1417]
The language picker button in the header navigation is missing an `aria-label`.

**Comment by moloko** [comment]
done in https://github.com/adaptlearning/adapt-contrib-languagePicker/commit/61890ec3415f1c3271963af37804710a85e97b14
